### PR TITLE
Lab3 personal solution

### PR DESCRIPTION
**Pull request by StruzzoStorpio**

### CODE REVIEW

**Review by bardhprenkaj — APPROVED** [review]
You're downloading the train set of MNIST and then splitting this subdataset into train and test set. There's a missing evaluation on the real test set of the MNIST. There's no hyperparameter optimization at all; and your hidden layers contain the same amount of neurons.
